main.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"io"
+	"net/http"
+	"strconv"
+
+	"encoding/json"
```

Comment:
all dependencies that are part of go library should be before the blank line

Reply:
ok